python/social_bot/models/wood_cube_10cm_without_offset/model.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 
 <model>
-  <name>Wood cube 10cm</name>
+  <name>Wood cube 10cm </name>
```

Comment:
Is this space after the name intentional?

Reply:
sorry, accidentally, not intentional... I'll remove it